### ISSUE TITLE
[Accessibility Bug] DDG needs visible focus outlining (WCAG 2.4.7)

### ISSUE THREAD

**Issue by charmarkk** [duckduckgo/duckduckgo-styles#69]
Copied from a previous Reddit post -- it took me far too long to find DDG's actual GitHub and actually fin a place to actually report bugs.

Please move if this is not the right repo, but this is explicitly related to CSS. I have hacked together a solution that only kind of works, and ultimately does not solve the problem.

My browser hack:
![image](https://user-images.githubusercontent.com/22264269/161833083-b3822111-7c43-43c4-a85b-898244b586b6.png)

___

I use a keyboard to navigate my computer. I've noticed that DDG:

-     doesn't have visual focus indicators (on anything, really)
-     only inconsistently lets me tab to search results (if I watch the URL preview on the bottom of the browser)
-     menus have odd behavior (don't open/close on keyboard focus or losing focus)

It's really hard to tell if there's any spots to report feedback like this, other than Reddit, unfortunately. Are there any devs I could work with or otherwise available spaces to discuss and work on these issues with?

Even Chrome can't display it's browser default, which tells me that visual focus is explicitly disallowed somewhere.

I love using DDG but, sorry, Google has ya beat here. These are simple issues, [and is required by WCAG.](https://www.w3.org/WAI/WCAG21/Understanding/focus-visible.html) Privacy and accessibility do not, should not, and cannot be mutually exclusive - we already get the short end of the stick online. Even worse, there is no proper method to convey problems like this, or any accessibility contact information to be found on the site.

I have since found out that there are keyboard shortcuts provided by DDG, but:

The shortcuts help (a lot, stylistically, visually), but that's not the default navigation expectation for keyboard users (hence my ignorance of the feature). As users decide to come to DDG from other search providers or browsers, they are not oriented to that, and won't go to search out a custom solution - they will expect the web-standard defaults to work. For me, it is MUCH more uncomfortable, and an unnecessary hurdle, to first tap M then use up and down arrows, when the tab key is a one stop shop for the entirety of the web - and requires no custom maintenance or JS!!

Keyboard shortcut styling:
![image](https://user-images.githubusercontent.com/22264269/161833245-aa14c95f-5eaa-4736-84ed-6c17e5c3bd91.png)

(This looks and makes so much more sense. It could be more contrasty, but is a step in the right direction.)

I'll have to remember that. In my many years of using DDG I not once learned about that, especially since I was drawn from Firefox's omnibox, not the DDG homepage or help documents.

**The focus indication provided by the custom solution could be/should be the default** - that'd be even better than most big search giants who just do underlining.

ETA: I never knew (or forgot) about the expanded theme options, either. However, they all also lack visual focus using default behavior, even the "contrast" theme, which is the one I'd have suspected included it.
